core/tests/protractor_utils/forms.js

```diff
@@ -419,8 +437,7 @@ var MultiSelectEditor = function(elem) {
       expect(actualSelection).toEqual(expectedCurrentSelection);
 
       // Close the dropdown menu at the end.
-      await elem.element(by.css(
-        '.protractor-test-search-bar-dropdown-toggle')).click();
+      await action.click('Test Dropdown Toggle', dropDownToggleButton);
     }
   };
 };
```

Comment:
ditto for the `.getText()`, `.getAttribute()`, and `.getTagName()` calls below

Reply:
updated!